src/main/kotlin/no/nav/familie/ef/sak/infrastruktur/config/CacheConfig.kt

```diff
@@ -40,15 +40,25 @@ class CacheConfig {
         }
     }
 
-    @Bean("oppgaveCache")
-    fun oppgaveCache(): CacheManager = object : ConcurrentMapCacheManager() {
+    @Bean("shortCache")
```

Comment:
Trenger ikke en egen cacheManager for oppgave, men kan bruke default

Reply:
Er vel mer at 10 minutter er litt "kort" cache på de mappene - som sikkert endres én gang i året... 🤷

Reply:
Endret sånn att mapper bruker vanlig cache som er 60 min